com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -420,6 +420,7 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
 
                 this.DisableAllKeywords(cmd);
                 context.ExecuteCommandBuffer(cmd);
+                cmd.Clear();
```

Comment:
And here. I think we should try to get rid of as many as we can. 
Again, it's ok to do in another PR with proper testing and time.
If we need to call it for some reason, we should try to come up with some basic rules when it's necessary to guide other devs and users.